Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -290,7 +290,7 @@ function Main {
 				}
 
 				$temp=(Get-Job -Id $TestJob).State
-					Write-LogInfo "-FINALLY-------------------------------------------------------------------$temp-------------------------"
+					Write-LogInfo "--------------------------------------------------------------------$temp-------------------------"
```

Comment:
Indent

Reply:
fixed in 66ae117079544b049a7a20e2d236a40d7b8e8825